src/sparseml/transformers/finetune/data/data_args.py

```diff
@@ -17,7 +17,37 @@
 
 
 @dataclass
-class DataTrainingArguments:
+class CustomDataTrainingArguments:
+    """
+    Arguments for training using custom datasets
+    """
+
+    data_files: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": """
+            The data in a dict format to be loaded using HF's load_dataset
+
+            Ex.
+            {
+                "train": [
+                    "path/to/folder/data1.json", # or csv
+                    "path/to/folder/data2.json",
+                ],
+                "test": ...
+            }"
+            """
+        },
+    )
+
+    text_column: Optional[str] = field(
+        default=None,
+        metadata={"help": "For custom datasets only. The text field key"},
+    )
+
```

Comment:
let's add an option for a preprocessing template - we should sync in slack or meet over this